### DISCUSSION TITLE
Hope to be able to customize the model to add smart search function/希望能够自定义添加智能搜索的模型的功能

### DISCUSSION THREAD

**Discussion by shenyan-008** [immich-app/immich#12827] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hope to be able to customize the model to add smart search function/希望能够自定义添加智能搜索的模型的功能

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Can you elaborate on what you're asking for exactly?
